connectors/github-pr/src/main/java/com/vmware/connectors/github/pr/v3/PullRequest.java

```diff
@@ -0,0 +1,332 @@
+package com.vmware.connectors.github.pr.v3;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.pojomatic.Pojomatic;
+import org.pojomatic.annotations.AutoProperty;
+
+import java.util.Date;
+
+/**
+ * https://developer.github.com/v3/pulls/
```

Comment:
I chose to implement a subset of the data that comes back.  I left in things that looked like they might be useful, even if I'm not using them all anymore (I went back and forth on what I would put into the cards).  What I'm sending back in the cards is a little overkill still, however, this code is really just an example for what our client might want to do.

If Dan or Marshall-Anne want me to adjust this, I can do it in followup tickets.